### ISSUE TITLE
new install fails

### ISSUE THREAD

**Issue by macmanes** [dib-lab/dammit#156]
```
conda env create -f environment.yml -n dammit-env
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata (repodata.json): done
Solving environment: \ WARNING conda.resolve:_get_sat_solver_cls(49): Could not run SAT solver through interface 'pycryptosat'.
\
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                                                                                                                                                                              -

UnsatisfiableError: The following specifications were found to be incompatible with each other:



Package zlib conflicts for:
khmer[version='>=2.1'] -> python=3.5 -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
numexpr[version='>=2.3.1'] -> python=3.6 -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
pip -> python[version='>=2.7,<2.8.0a0'] -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> glib[version='>=2.56.2,<2.57.0a0'] -> zlib=1.2
pandas -> python[version='>=3.6,<3.7.0a0'] -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
last -> python=3.6 -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8']
python -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
last -> zlib[version='>=1.2.11,<1.3.0a0']
pytest-runner -> python=3.5 -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
pytest -> python[version='>=3.6,<3.7.0a0'] -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> zlib[version='1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
numpy -> python[version='>=2.7,<2.8.0a0'] -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> glib[version='>=2.56.2,<2.57.0a0'] -> python[version='>=3.7,<3.8.0a0'] -> zlib=1.2.11
busco=3.0.2 -> augustus[version='>=3.2.3'] -> zlib[version='1.2.*|1.2.8.*|1.2.8']
doit[version='>=0.29.0'] -> python=2.7 -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> dbus[version='>=1.13.6,<2.0a0'] -> glib[version='>=2.58.3,<3.0a0'] -> zlib[version='1.2.*|1.2.11|1.2.8']
matplotlib -> python[version='>=3.6,<3.7.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']
sphinx_rtd_theme[version='>=0.1.9'] -> python[version='>=3.6,<3.7.0a0'] -> zlib[version='1.2.*|1.2.11.*|1.2.11|1.2.8|>=1.2.11,<1.3.0a0']
transdecoder=3.0.1 -> cd-hit -> zlib[version='>=1.2.11,<1.3.0a0']
Package sqlite conflicts for:
last -> python=3.6 -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> glib[version='>=2.56.2,<2.57.0a0'] -> python[version='>=3.7,<3.8.0a0'] -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
sphinx_rtd_theme[version='>=0.1.9'] -> python[version='>=3.6,<3.7.0a0'] -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
pandas -> python[version='>=3.6,<3.7.0a0'] -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
doit[version='>=0.29.0'] -> python=2.7 -> sqlite[version='3.13.*|3.20.*|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0']
numexpr[version='>=2.3.1'] -> python=3.6 -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
python -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
numpy -> python[version='>=2.7,<2.8.0a0'] -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
matplotlib -> python[version='>=3.6,<3.7.0a0'] -> sqlite[version='>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> sqlite[version='>=3.29.0,<4.0a0']
pytest-runner -> python=3.5 -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
busco=3.0.2 -> python[version='>=3.5,<3.6.0a0'] -> sqlite=3.13
pip -> python[version='>=2.7,<2.8.0a0'] -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
khmer[version='>=2.1'] -> python=3.5 -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
pytest -> python[version='>=3.6,<3.7.0a0'] -> sqlite[version='3.13.*|3.20.*|3.9.*|>=3.24.0,<4.0a0|>=3.25.1,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.28.0,<4.0a0|>=3.30.1,<4.0a0']
Package python conflicts for:
busco=3.0.2 -> python[version='3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
python
pytest -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
pytest-runner -> pytest -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
doit[version='>=0.29.0'] -> python=2.7
busco=3.0.2 -> augustus[version='>=3.2.3'] -> boost=1.64 -> python[version='2.7.*|3.4.*']
last -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
pytest-runner -> python[version='2.7.*|3.4.*|3.5.*|3.6.*']
pip -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
pandas -> python-dateutil[version='>=2.5.*'] -> python
matplotlib -> matplotlib-base==2.2.3=py36h60b886d_1 -> cycler[version='>=0.10'] -> python[version='<3']
numexpr[version='>=2.3.1'] -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
sphinx_rtd_theme[version='>=0.1.9'] -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0']
numpy -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
khmer[version='>=2.1'] -> bz2file -> python
matplotlib -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
doit[version='>=0.29.0'] -> configparser -> python[version='>=2.7,<2.8.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> glib[version='>=2.56.2,<2.57.0a0'] -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
khmer[version='>=2.1'] -> python[version='2.7.*|3.5.*|3.6.*|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
Package harfbuzz conflicts for:
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> harfbuzz[version='>=1.9.0,<2.0a0|>=2.3.0,<3.0a0|>=2.4.0,<3.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> pango=1.40 -> harfbuzz[version='1.3.*|1.4.*|1.7.*|>=1.7.6,<1.8.0a0|>=1.8.4,<2.0a0|>=1.8.6,<2.0a0|>=1.9.0,<2.0a0|>=2.3.0,<3.0a0|>=2.4.0,<3.0a0']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> pango[version='>=1.40.14,<1.41.0a0'] -> harfbuzz[version='1.7.*|>=1.7.6,<1.8.0a0|>=1.8.4,<2.0a0|>=1.8.6,<2.0a0']
busco=3.0.2 -> r-base -> pango=1.40 -> harfbuzz[version='1.3.*|1.4.*']
Package ncurses conflicts for:
matplotlib -> python[version='>=3.6,<3.7.0a0'] -> ncurses[version='>=6.1,<6.2.0a0']
last -> python=3.6 -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
khmer[version='>=2.1'] -> python=3.5 -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
numexpr[version='>=2.3.1'] -> python=3.6 -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
pytest-runner -> python=3.5 -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
pip -> python[version='>=2.7,<2.8.0a0'] -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
pandas -> python[version='>=3.6,<3.7.0a0'] -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
doit[version='>=0.29.0'] -> python=2.7 -> ncurses[version='5.9.*|>=6.1,<6.2.0a0']
pytest -> python[version='>=3.6,<3.7.0a0'] -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
doit[version='>=0.29.0'] -> python=2.7 -> sqlite=3.13 -> ncurses==5.9
busco=3.0.2 -> python[version='>=3.5,<3.6.0a0'] -> ncurses=5.9
python -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
sphinx_rtd_theme[version='>=0.1.9'] -> python[version='>=3.6,<3.7.0a0'] -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
numpy -> python[version='>=2.7,<2.8.0a0'] -> ncurses[version='5.9|5.9.*|>=6.1,<6.2.0a0']
Package freetype conflicts for:
last -> pillow -> freetype[version='2.6.*|2.7|2.7|2.7.*']
busco=3.0.2 -> r-base -> pango=1.40 -> freetype[version='2.6.*|2.7.*|2.7|2.7.*']
matplotlib -> matplotlib-base==2.2.3=py36h60b886d_1 -> freetype[version='>=2.9.1,<3.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> cairo[version='>=1.14.12,<1.15.0a0'] -> freetype[version='2.6.*|2.7.*|2.7|2.8.1|2.7|2.7.*|>=2.8.1,<2.9.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> freetype[version='>=2.9.1,<3.0a0']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> cairo[version='>=1.16.0,<1.17.0a0'] -> freetype[version='2.6.*|2.7|2.8.1|2.7|2.7.*|>=2.8.1,<2.9.0a0']
busco=3.0.2 -> r-base -> pango=1.40 -> fontconfig=2.12 -> freetype==2.7
Package libxml2 conflicts for:
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> libxml2[version='>=2.9.9,<2.10.0a0']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> fontconfig[version='>=2.13.1,<3.0a0'] -> libxml2[version='>=2.9.8,<2.10.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> libxml2[version='2.9.*|>=2.9.8,<2.10.0a0']
transdecoder=3.0.1 -> cd-hit -> openmp -> clangdev[version='5.0.0|5.0.0.*'] -> libxml2[version='>=2.9.8,<2.10.0a0']
busco=3.0.2 -> r-base -> libxml2=2.9
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> pango=1.40 -> fontconfig[version='>=2.13.1,<3.0a0'] -> libxml2[version='>=2.9.9,<2.10.0a0']
busco=3.0.2 -> r-base -> pango=1.40 -> fontconfig=2.12 -> libxml2
Package libpng conflicts for:
busco=3.0.2 -> r-base -> pango=1.40 -> libpng[version='>=1.6.21,<1.7|>=1.6.23,<1.7']
busco=3.0.2 -> r-base -> pango=1.40 -> freetype=2.7 -> libpng[version='1.6.*|>=1.6.22,<1.6.31']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> cairo[version='>=1.16.0,<1.17.0a0'] -> libpng[version='>=1.6.22,<1.6.31|>=1.6.23,<1.7|>=1.6.28,<1.7|>=1.6.32,<1.6.35']
matplotlib -> matplotlib-base==2.2.3=py36h60b886d_1 -> libpng[version='>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0']
last -> pillow -> freetype=2.6 -> libpng[version='1.6.*|>=1.6.21,<1.7|>=1.6.22,<1.6.31|>=1.6.23,<1.7|>=1.6.28,<1.7|>=1.6.35,<1.7.0a0']
matplotlib -> matplotlib-base==2.2.3=py36h60b886d_1 -> freetype[version='>=2.9.1,<3.0a0'] -> libpng[version='>=1.6.34,<1.7.0a0']
busco=3.0.2 -> r-base -> libpng[version='>=1.6.28,<1.7']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> pango=1.40 -> libpng[version='1.6.*|>=1.6.21,<1.7|>=1.6.23,<1.7|>=1.6.37,<1.7.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> libpng[version='>=1.6.22,<1.6.31|>=1.6.28,<1.7|>=1.6.32,<1.6.35|>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0']
Package readline conflicts for:
pytest-runner -> python=3.5 -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
doit[version='>=0.29.0'] -> python=2.7 -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
numpy -> python[version='>=2.7,<2.8.0a0'] -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
python -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
pandas -> python[version='>=3.6,<3.7.0a0'] -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
pytest -> python[version='>=3.6,<3.7.0a0'] -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
numexpr[version='>=2.3.1'] -> python=3.6 -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> glib[version='>=2.56.2,<2.57.0a0'] -> python[version='>=3.7,<3.8.0a0'] -> readline[version='7.0.*|>=8.0,<9.0a0']
sphinx_rtd_theme[version='>=0.1.9'] -> python[version='>=3.6,<3.7.0a0'] -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
khmer[version='>=2.1'] -> python=3.5 -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
pip -> python[version='>=2.7,<2.8.0a0'] -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> readline[version='6.2.*|7.0|>=7.0,<8.0a0']
busco=3.0.2 -> python[version='>=3.5,<3.6.0a0'] -> readline=6.2
last -> python=3.6 -> readline[version='6.2.*|7.0.*|7.0|>=7.0,<8.0a0|>=8.0,<9.0a0']
matplotlib -> python[version='>=3.6,<3.7.0a0'] -> readline[version='>=7.0,<8.0a0|>=8.0,<9.0a0']
Package tk conflicts for:
doit[version='>=0.29.0'] -> python=2.7 -> tk[version='8.5.*|8.6.*|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
pandas -> python[version='>=3.6,<3.7.0a0'] -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
matplotlib -> python[version='>=3.6,<3.7.0a0'] -> tk[version='>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
numexpr[version='>=2.3.1'] -> python=3.6 -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
sphinx_rtd_theme[version='>=0.1.9'] -> python[version='>=3.6,<3.7.0a0'] -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
pytest -> python[version='>=3.6,<3.7.0a0'] -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
python -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
khmer[version='>=2.1'] -> python=3.5 -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
last -> python=3.6 -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> krb5[version='>=1.16.2,<1.17.0a0'] -> tk[version='>=8.6.7,<8.7.0a0']
busco=3.0.2 -> python[version='>=3.5,<3.6.0a0'] -> tk=8.5
pip -> python[version='>=2.7,<2.8.0a0'] -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
numpy -> python[version='>=2.7,<2.8.0a0'] -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
pytest-runner -> python=3.5 -> tk[version='8.5.*|8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> tk[version='8.5.*|8.6.*|>=8.6.8,<8.7.0a0|>=8.6.9,<8.7.0a0']
Package fontconfig conflicts for:
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> fontconfig[version='>=2.13.1,<3.0a0']
busco=3.0.2 -> r-base -> pango=1.40 -> fontconfig[version='2.11.*|2.12.*']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> cairo[version='>=1.16.0,<1.17.0a0'] -> fontconfig[version='2.11.*|2.12.*|>=2.13.0,<3.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> pango=1.40 -> fontconfig[version='2.11.*|2.12.*|>=2.13.0,<3.0a0|>=2.13.1,<3.0a0']
Package pango conflicts for:
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> pango[version='>=1.40.14,<1.41.0a0|>=1.42.4,<1.43.0a0']
bioconductor-seqlogo -> r-base[version='>=3.4.1,<3.4.2.0a0'] -> pango[version='1.40.*|>=1.40.14,<1.41.0a0']
busco=3.0.2 -> r-base -> pango=1.40
Package perl-io-zlib conflicts for:
busco=3.0.2 -> blast -> perl-archive-tar -> perl-io-zlib
Package libclang conflicts for:
transdecoder=3.0.1 -> cd-hit -> openmp -> clangdev[version='5.0.0|5.0.0.*'] -> libclang[version='8.0.0|8.0.1|8.0.1',build='hc9558a2_0|hc9558a2_6|hc9558a2_5|hc9558a2_4|hc9558a2_1']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> libclang[version='>=9.0.0,<10.0a0']
Package atk conflicts for:
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2 -> atk[version='>=2.32.0,<3.0a0']
Package gtk2 conflicts for:
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> qt[version='>=5.12.5,<5.13.0a0'] -> gtk2
Package python-dateutil conflicts for:
pandas -> python-dateutil[version='>=2.5.*|>=2.6.1']
matplotlib -> pyqt[version='>=5.6.0,<5.7.0a0'] -> python-dateutil
Package perl-compress-raw-zlib conflicts for:
busco=3.0.2 -> blast -> perl-archive-tar -> perl-io-compress -> perl-compress-raw-zlib[version='>=2.084|>=2.087']
Note that strict channel priority may have removed packages required for satisfiability.

**Comment by camillescott** [comment]
Hi @macmanes -- not sure what `environment.yml` contains. The current bioconda version should install properly in a clean environment, and fixes some major bugs as well.

**Comment by bluegenes** [comment]
> Hi @macmanes -- not sure what `environment.yml` contains. The current bioconda version should install properly in a clean environment, and fixes some major bugs as well.

follow-up: that was an environment that installed dammit from the github master branch, but apparently failed for the same issues that were fixed in the update to 1.2. Bioconda install should be working now!
